src/modules/edit.test.js

```diff
@@ -0,0 +1,13 @@
+const editTask = require('./edit.js');
+
+test('should edit task', () => {
+  expect(() => {
+    editTask({
+      index: 1,
+      description: 'Read book',
+      completed: true,
+    });
+  }).toHaveLength(0);
```

Comment:
- [ ] I think it's better to test if the function is editing a task. Check if the description has the input value after running the function. Your function performs changes on your local storage. Use mock instead of your local storage. You have to check if the changes are working. The knowledge of performing tests is very important to every developer's career.
- [ ] Please group your tests using the description method. Your tests will look more professional for recruiters and hiring managers.